documentation/final-report/tex/evaluation.tex

```diff
@@ -3,19 +3,18 @@ \section{Evaluation of the functional modules, the product, and the failure anal
-We also implemented all could-have requirements. Overall, we are happy with the end-result. We are really only missing one feature that was in the initial plan, and what we have works well and looks really good too.
+This has as effect that we also did not complete the should-have requirement "Warnings for possible problems when the director edits the timeline during recording, such as collisions". At first, we implemented automatic timeline speed (so that the shot caller did not have to press the button every time), but feedback from PolyCast indicated that this functionality does not work in practice, as the speed of the concert is never exactly known and/or stable. Therefore, we stayed with manual advancing of the timeline. Instead of highlighting the cameras that a certain camera operator is operating, we have created a user selection screen in which a camera operator can choose which cameras he operates. Then, only these cameras will be shown. This way, we minimize the redundant information on the screen for any given camera operator.\\
+We also implemented all could-have requirements. Overall, we are satisfied with the end-result. We are really only missing one feature that was in the initial plan, and what we have works well and looks really good too.
 
 \subsection{The camera control center}
 Unfortunately, this is the part that is finished the least. It is possible to create presets, and recall them, as well as moving the cameras around. However, this was not yet linked with the web application, so the automatic recall functionality is not there at this point. Unfortunately, we aimed a little too high at the start of the project, creating three fully functional modules. Because of that, we shifted focus on getting the scripting application and live view really done, because we rather had two great applications than three half-finished ones.\\
 Looking at the initial requirements, we're missing the two must-haves that have to do with automatic recalling. We were able to get a low-quality preview of the camera you are operating, and the possibility to move/zoom the cameras.\\
-Possible improvements in this part will be discussed in the next chapter.
+We concluded that this module was no close to finished. Because of this, and more importantly since an other group implemented a preset generation interface that was much more sophisticated than ours, we decided not to put it in our final system, but instead work together with the other group. This co-operation with this group, called BeNine, was recommended by our supervisors, and would greatly improve the strength of the entire workflow. Further explanation on this will be in the next chapter.
```

Comment:
A few grammar tips:
- `not close` instead of `no close`
- `another` instead of `an other`